overlay/etc/dnsmasq.d/steamcache.conf

```diff
@@ -0,0 +1,55 @@
+# Run as root to make things easy.
+user=root
+group=root
+
+# Add Google's public DNS servers as a fallback.
```

Comment:
Yeah, I agree this should be a Docker flag for all of the steamcache projects. I've already made the change on a branch and it works fine. nginx doesn't like the lack of an explicit nameserver but that's another repo and another pull request.